document_merge_service/api/engines.py

```diff
@@ -35,6 +56,20 @@ class DocxMailmergeEngine(DocxValidator):
     def __init__(self, template):
         self.template = template
 
+    def _has_placeholders(self, part):
+        return bool(part.findall("//MergeField"))
+
+    def validate_template_syntax(self):
+        document = MailMerge(self.template)
+        # syntax can't be invalid as it's validated by office
+        # suites. However we need to have *some* placeholders
```

Comment:
Do we really must have placeholders? I know, it doesn't make sense to use MailMerge without them. But do we know it's never used that way so far?

Reply:
I assume that if you want to use the functionality, you also intend to have some placeholders. We could also leave out validation for this type of templates, but this way, we could also detect possible mismatch of the engine type.

Reply:
How is the syntax validated by office suites?